crm/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import include, url
+from rest_framework import routers
+from . import views
+
+router = routers.DefaultRouter()
+router.register(r'ViewCompanies', views.ViewCompanies)
+
+# Wire up our API using automatic URL routing.
+# Additionally, we include login URLs for the browsable API.
+urlpatterns = [
+    url('', include(router.urls)),
+    url('api-auth/', include('rest_framework.urls', namespace='rest_framework'))
+]
```

Comment:
If you could optimize the code to maybe follow these guidelines it would give us a standard way of how we are developing API routes.
[Styleguide url](https://github.com/HackSoftware/Django-Styleguide#urls)